test/lit/parse-nominal-types-extends.wast

```diff
@@ -1,62 +1,66 @@
 ;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
 
 ;; Test that new-style nominal types are parsed correctly.
-;; TODO: Remove --nominal below once nominal types are parsed as nominal by default.
 
-;; RUN: foreach %s %t wasm-opt --nominal -all -S -o - | filecheck %s
-;; RUN: foreach %s %t wasm-opt --nominal -all --roundtrip -S -o - | filecheck %s
+;; RUN: foreach %s %t wasm-opt -all -S -o - | filecheck %s
+;; RUN: foreach %s %t wasm-opt -all --roundtrip -S -o - | filecheck %s
 
 ;; void function type
 (module
-  (type $sub (func) (extends $super))
-
   ;; CHECK:      (type $super (func))
   (type $super (func))
 
-  ;; CHECK:      (global $g (ref null $super) (ref.null nofunc))
-  (global $g (ref null $super) (ref.null $sub))
+  ;; CHECK:      (type $sub (func_subtype $super))
+  (type $sub (func) (extends $super))
+
+  ;; CHECK:      (global $g (ref null $sub) (ref.null nofunc))
+  (global $g (ref null $sub) (ref.null nofunc))
```

Comment:
I understand why the types are reordered, but why is the global's type changed?

Reply:
I noticed that `$sub` wasn't appearing in the output, which was probably an unintended consequence of changing `ref.null` to use bottom types. Changing the type of the global ensures both types appear in the output.